<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_BluetoothInfiShow">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BluetoothName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BluetoothDevice">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SendData">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="on_pushButton_help_clicked">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_OpenBluetooth">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_BluetoothScan">
       <property name="text">
        <string>扫描</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_StopScan">
       <property name="text">
        <string>停止扫描</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CloseBluetooth">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
